demosplan/DemosPlanCoreBundle/Controller/Statement/DemosPlanStatementTagController.php

```diff
@@ -180,8 +182,14 @@ public function tagListEditAction(
                     $result = $statementHandler->createTagFromTopicId($requestPost['r_createtags'], $newTag, $procedure);
                     $this->getMessageBag()->add('confirm', 'confirm.tag.created');
                     $anchor = $result->getId();
-                } catch (Exception) {
-                    $this->getMessageBag()->add('warning', 'warning.tag.created');
+                } catch (DuplicatedTagTitleException $e) {
+                    $this->getMessageBag()->add('error', 'error.import.tag.name.taken', ['tagTitle' => $e->getTagTitle(), 'topicName' => $e->getTopic()->getTitle()]);
+                } catch (TagTopicNotFoundException $e) {
+                    $this->getMessageBag()->add('error', 'error.topic.notfound');
+                    $this->logger->error('error.topic.notfound', [$e]);
+                } catch (Exception $e) {
+                    $this->getMessageBag()->add('error', 'error.tag.add');
+                    $this->logger->error('error.tag.add', [$e]);
```

Comment:
It doesn't make much difference, but why did you replace `warning.tag.created` with `error.tag.add`? The former one was a bit more specific.

Reply:
i did adjust it hier https://github.com/demos-europe/demosplan-core/pull/2018/commits/07c23fa1636376e951a988a945121b0443643e70, bit i think this an error message and not a warning ! in this case the tag will be not created while an error ( other than DuplicatedTagTitleException and TagTopicNotFoundException ) occurred